synedrion/src/cggmp21/params.rs

```diff
@@ -164,45 +203,80 @@ pub struct TestParams;
 // - Range checks will fail with the probability $q / 2^\eps$, so $\eps$ should be large enough.
 // - P^{fac} assumes $N ~ 2^{4 \ell + 2 \eps}$
 impl SchemeParams for TestParams {
+    type Curve = TinyCurve64;
+    type WideCurveUint = bigintv05::U384;
+    // TODO: 8*24 = 192, this is to work around an issue with the ModulusSize-trait. This should be ideally be 8 bytes long.
+    type HashOutput = [u8; 24];
     const SECURITY_BITS: usize = 16;
     const SECURITY_PARAMETER: usize = 10;
     const L_BOUND: u32 = 256;
```

Comment:
These bounds can be amended now that we have a small-order curve. l = log2(q) (so 32 or 64), lp = l * 5, eps = l * 2

Reply:
And the `PaillierTest` uint sizes adjusted accordingly.

Reply:
Finally indeed! I played with it a little bit last night but ended up stashing the changes because it just made the review even bigger. Follow up PR yes?

Reply:
Added https://github.com/entropyxyz/synedrion/issues/192